packages/metablocks/src/api.ts

```diff
@@ -1,27 +1,19 @@
-import { getMetaBlocksProgram } from "./factory";
+import { getMetaBlocksProgram } from './factory';
 import {
   computeCreateUniverseParams,
   computeGroupedDepositNftParams,
   computeUpdateUniverseParams,
-} from "./paramsBuilder";
-import { Transaction } from "@solana/web3.js";
+} from './paramsBuilder';
+import { Transaction } from '@solana/web3.js';
+import { getWithdrawNftInstruction } from './instructions';
 import {
-  getDepositNftInstruction,
-  getInitDepositNftInstruction,
-  getInitReceiptMintInstruction,
-  getTransferReceiptNftToUserInstruction,
-  getWithdrawNftInstruction,
-} from "./instructions";
-import {
-  DepositNftApiArgs,
+  FetchAccountArgs,
   GroupedDepositNftApiArgs,
-  InitDepositNftApiArgs,
-  InitReceiptMintApiArgs,
   SendTxRequest,
-  TransferReceiptNftApiArgs,
   UniverseApiArgs,
   WithdrawNftApiArgs,
-} from "./types/types";
+} from './types/types';
+import { accounts_api } from '.';
```

Comment:
What is happening here? 

1. Why are we importing from `.` ?
2. Why are we not using camelCase like `accountsApi`? Both of the file name and import?